### ISSUE TITLE
Hierarchical reports exports empty datasets on leafs

### ISSUE THREAD

**Issue by parruc** [matomo-org/matomo#13160]
When selecting "Row evolution" of a hierarchical report (IE download report) The CSV export function works on groups but not on "leafs" of the hierarchy. Specifically it exports an empty dataset.

Also reported on the forum by another user with screenshots here:

https://forum.matomo.org/t/report-has-data-but-export-is-empty/28094

**Comment by sgiehl** [comment]
@parruc  I'm not able to reproduce that with latest RC. Are you able to reproduce that on our demo (https://demo.matomo.org).  Maybe it was fixed meanwhile?

**Comment by parruc** [comment]
@sgiehl Yes, I was able to reproduce it on the demo installation. Follow the following steps:

- Select virtual-drums.com website
- Go to behavior -> pages, you should see the hierarchical report
- expand photos.php and screenshots-virtual-drums-1
- click on open row evolution of the index.php hierarchy leaf inside screenshots-virtual-drums-1
- export the open row evolution as csv or xml

It should export an empty csv or xml

**Comment by sbruemmer** [comment]
@sgiehl: I was able to reproduce as well under https://demo.matomo.org:

1. Select "Matomo Forums"
2. Go to Behaviour > Pages
3. Expand "t"
4. Open "Row Evolution" of any subpage
5. export as csv or xml

It produces an empty file.

![subpages](https://user-images.githubusercontent.com/13978063/47794048-f2adc800-dd1f-11e8-9dc6-04e0782822e7.jpg)

**Comment by t4b4sc0** [comment]
@sgiehl we have the same problem with our installation, latest matomo version.

**Comment by DonCarlosGraz** [comment]
same her

**Comment by parruc** [comment]
Three confirmation to my original issue all from different people. May be the label "Waiting for user feedback / Cannot reproduce yet  label" should be removed
